docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -6,6 +6,66 @@ Use this extension to get the full list of problems found by Codacy for a pull r
 
 !!! info "To use this extension you need a [Codacy account](https://www.codacy.com/signup-codacy)"
 
+## Interface overview
+
+The main view of the extension displays information about the code quality changes introduced by the pull request you're working on, as well as the status of other open pull requests and analyzed branches. This information is grouped into three tabs:
+
+-   [Status](#status-tab)
+-   [Open Pull Requests](#open-pull-requests-tab)
+-   [Analyzed branch](#analyzed-branch-tab)
+
+![Important issues](./images/codacy-vscode-extension-main-view.png)
+
+### Status tab
+
+The pull request **Status** tab displays the following information for the currently checked out pull request:
```

Comment:
I initially used something similar to what you suggested, but GitHub seems OK with using "check out a PR" on [their docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally), and I liked the idea of shortening the sentence.

If you think it may confuse our users or find it confusing yourself, I'm happy to use your variant.

Reply:
For users like me who might not be completely comfortable with git operations, I would prefer to be technically accurate. 🤔 
If checking out a branch is different from checking out a pull request, let's use the correct term for the operation Codacy is executing. On the other hand, if it's technically the same, then I believe it's ok to use either.
Thoughts?